src/store/activities/reducer.test.js

```diff
@@ -80,12 +77,12 @@ describe('activities reducer', () => {
     expect(state).toEqual(state);
   });
 
-  it('should set expenses', () => {
```

Comment:
tooooooo much making the copies instead of writing it by your own ! 😠

Reply:
:( just to explain myself, I REALLY don't copy stuff from somewhere else, it would be a waste of my and your time. Sometimes when I'm stuck I search for answers in this React Udemy course and as it is concentrated on making this EXPENSES app and I really try to understand what they are doing and I focus extremely I catch myself doing stupid mistakes like this. Implementing things I just understood often I just rush with the flow and it ends up like this :/